components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -151,247 +194,400 @@ message TimeQuery {
 	repeated Support supports = 16;
+	// Other profiles that this profile depends on
 	repeated Dependency depends = 17;
+	// A unique value generated from the profile used to identify it
 	string sha256 = 18;
+	// ???
 	repeated Group groups = 19;
+	// The controls defined on the profile
 	repeated Control controls = 20;
+	// The attributes defined on the profile
 	repeated Attribute attributes = 21;
+	// The highest version number of the profile stored in Automate
 	string latest_version = 22;
+	// The status of the profile in the generated report
 	string status = 23;
+	// The reason this profile was skipped in the generated report, if any
 	string skip_message = 24;
 }
 
 message Ref {
+	// The URL of the external document
 	string url = 1;
+	// The description of the external document
 	string ref = 2;
 }
 
 message Result {
+	// The status of the test
 	string status = 1;
+	// The description of the test
 	string code_desc = 2;
+	// The time taken to run the test
 	float run_time = 3;
+	// The timestamp of when this individual test was run
 	string start_time = 4;
+	// The reason the test failed, if any
 	string message = 5;
+	// The reason the test was skipped, if any
 	string skip_message = 6;
 }
 
 message SourceLocation {
+	// The source code file the control is defined in
 	string ref = 1;
+	// The line number the control is defined on
 	int32 line = 2;
 }
 
 message Option {
+	// The description of the attribute
 	string description = 1;
+	// The default value of the attribute
 	string default = 2 ;
 }
 
 message Support {
+	// The name of the supported operating system
 	string os_name = 1;
+	// The name of the broader category of the supported platform (eg, linux, windows)
 	string os_family = 2;
+	// The specific release of the operating system this profile supports
 	string release = 3;
+	// The supported inspec version this profile was made to run on
 	string inspec_version = 4;
+	// The platform name and version combined
 	string platform = 5;
 }
 
 message Dependency {
+	// The name of the profile
 	string name = 1;
+	// The URL of the profile accessible over HTTP or HTTPS
 	string url = 2;
+	// The path to the profile on disk
 	string path = 3;
+	// The git URL of the profile
 	string git = 4;
+	// The specific git branch of the dependency
 	string branch = 5;
+	// The specific git tag of the dependency
 	string tag = 6;
+	// The specific git commit of the dependency
 	string commit = 7;
+	// The specific git version of the dependency
 	string version = 8;
+	// The name of the dependency stored in Chef Supermarket
 	string supermarket = 9;
+	// The short name of the dependency stored on Github
 	string github = 10;
+	// The short name of the dependency stored on the Chef Automate or Chef Compliance server
 	string compliance = 11;
+	// The status of the dependency in the report
 	string status = 12;
+	// The reason this profile was skipped in the generated report, if any
 	string skip_message = 13;
 }
 
 message Group {
+	// ???
```

Comment:
uh tbh i don't remember what this means either...i am making a note to educate myself.